handlers/_swagger.js

```diff
@@ -403,15 +404,31 @@ function registerRoute(app, auth, additionalMiddleware, method, path, data, allo
                             'Response validation error:', JSON.stringify(validationErrors, null, 2)
                         );
                     }
+                    
+                    // attach the body for post-send() middlewares
+                    res.body = body;
```

Comment:
is this the right name to use?
would `sentBody` be better?
would attaching it later in the process, after the `res.send` is called, be better for readability?

Reply:
`sentBody` is fine, we'll change to that name.

We did it this way to avoid reparsing the stringified `sentBody`. It could be done the other way if you prefer?